src/eventHandlers/pullRequest/test.spec.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * @webhook-pragma pull_request_for_major_bump
```

Comment:
@viestat you're suggestion of creating a new test file did work although I'm not quite sure what I should call this file. Any suggestions?

Reply:
Renamed it to `majorBump.spec.ts` as @viestat suggested